src/data/user-monitoring/permission-fixer/PermissionFixerUserGroupD2Repository.ts

```diff
@@ -23,21 +24,45 @@ export class PermissionFixerUserGroupD2Repository implements PermissionFixerUser
             throw new UserGroupNotFoundException("Error getting user group: " + groupsIds);
         }
     }
-    async save(userGroup: PermissionFixerUserGroupExtended): Async<string> {
+    async save(userGroup: PermissionFixerUserGroupExtended, users: Ref[]): Async<string> {
         try {
-            const response = await this.api.models.userGroups.put(userGroup).getData();
-            if (response.status == "OK") {
+            const response = await this.appendUsersInUsergroup(userGroup, users);
```

Comment:
Pedantic note: "in" usually means current location, "to" or "into" are more fitting.
So it should be `appendUsersToUsergroup` to accurately describe what the function does.

Reply:
na it's not  pedantic, thanks!